tests/EpisodeIntegrationServiceTests/ReceiveDataTests/ReceiveDataTests.cs

```diff
@@ -84,10 +203,10 @@ public async Task ReceiveData_ShouldLogErrorOnFindingABadRowInEpisodesCsvFile()
             log.Log(
             LogLevel.Error,
             0,
-            It.Is<object>(state => state.ToString().Contains("Error in ProcessEpisodeDataAsync: ")),
+            It.Is<object>(state => state.ToString().Contains("Error in ReceiveData: ")),
             It.IsAny<Exception>(),
             (Func<object, Exception, string>)It.IsAny<object>()),
-            Times.Exactly(2));
+            Times.Once());
```

Comment:
Given the test name and the test data, I would expect this to be verifying that two errors were logged